app/database2.js

```diff
@@ -0,0 +1,58 @@
+let express = require("express");
+let { Pool } = require("pg");
+let env = require("E:/starter2/dbenv.json");
+const argon2 = require('argon2');
+
+let hostname = "localhost";
+let port = 3000;
+let app = express();
+
+app.use(express.json());
+
+let pool = new Pool(env);
+pool.connect().then(() => {
+    console.log("Connected to database");
+});
+
+app.post('/login', async (req, res) => {
+    let { username, password } = {username:'user11', password:'12345'};
+    /*
+    let body = req.body;
+    let { username, password } = body;  used when data is sent from frontend
+    */
+  
+    try {
+      let queryResult = await pool.query('SELECT * FROM userdata WHERE username = $1', [username]);
+      let user = queryResult.rows[0];
+      
+
+      const hashpassword = await argon2.hash(password);
```

Comment:
I don't think we need to hash the plaintext password unless we're creating a new account. I suggest moving this inside the `if(!user)` block below